library/std/src/sys/unix/process/process_unix.rs

```diff
@@ -490,8 +492,16 @@ impl ExitStatus {
         libc::WIFEXITED(self.0)
     }
 
-    pub fn success(&self) -> bool {
-        self.code() == Some(0)
```

Comment:
does this just straight up remove the `success` method or am I reading this wrong? I would think this is a major breaking change

Reply:
> does this just straight up remove the `success` method or am I reading this wrong? I would think this is a major breaking change

This is the *internal* `success` method.  The structure of this code is a bit confusing.  There is the public `ExitStatus` which is a newtype facade around an OS-specific `imp::ExitStatus`.

With those internal implementations gaining `.exit_ok()`, there is no need for a specific `imp::ExitStatus::success()` - the facade `ExitStatus::success()` is still there but is simply `.exit_ok().is_ok()`.